src/pages/clouddocs/annotator_concept_detection.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Concept Detection"
+excerpt: "."
+categories: Annotators
+slug: annotator_concept_detection
+toc: true
+---
+<!-- ---
+
+copyright:
+  years: 2011, 2021
+lastupdated: "2019-09-21"
+
+keywords: annotator clinical data, clinical data, annotation
+
+subcollection: wh-acd
+
+---
+
+# Concept Detection -->
+
+The concept detection service detects medical concepts from unstructured data. The service provides concepts based on the Unified Medical Language System (UMLS). As of the 2018AA version of the UMLS library, the consumers can elect to have a set of [medical codes](/docs/wh-acd?topic=wh-acd-medical_codes#medical_codes) associated with the UMLS concepts by specifying the optional configuration parameter to return the medical codes. When medical codes are requested, the UMLS concept annotations from concept detection will include the applicable medical codes as metadata within the annotations.
+
+## Expanded Concepts
+
+Concept detection provides an _expanded_ option that allows you to go beyond exact matching of surface forms in a dictionary. With **expanded** set to `true`, concept detection will learn from the dictionary entries you have specified and generalize to other ways that concepts can be expressed in text. This allows you to define a core set of surface forms without exhaustively listing every surface form for every concept. This becomes particularly important with compound ideas. Consider the following text:
+
+`The patient broke his leg and hip when he fell outside his home.`
+
+There are two different injuries expressed in this text that we want to capture - broken leg and broken hip.  Neither one will have a dictionary entry that covers the way the concept is expressed. In this example, concept detection would return the following concepts with **expanded** set `true`.
+
+```
+{
+    "cui": "C0159852",
+    "preferredName": "Fracture of tibia and fibula",
+    "semanticType": "inpo",
+    "source": "umls-expanded",
+    "sourceVersion": "2018AA",
+    "type": "umls.InjuryOrPoisoning",
+    "begin": 12,
+    "end": 25,
+    "coveredText": "broke his leg",
+    "icd10Code": "S82.90X?",
+    "snomedConceptId": "414293001",
+    "vocabs": "MTH,CHV,CCS,ICD9CM,SNOMEDCT_US,ICPC"
+}
+
+...
+
+{
+    "cui": "C0019557",
+    "preferredName": "Hip Fractures",
+    "semanticType": "inpo",
+    "source": "umls-expanded",
+    "sourceVersion": "2018AA",
+    "type": "umls.InjuryOrPoisoning",
+    "begin": 12,
+    "end": 33,
+    "coveredText": "broke his leg and hip",
+    "loincId": "MTHU020794",
+    "icd10Code": "S72.009?",
+    "nciCode": "C26794,C35153",
+    "snomedConceptId": "5913000,263225007",
+    "meshId": "M0010366",
+    "vocabs": "MTH,CHV,LNC,CSP,MSH,NCI,AOD,NCI_CTCAE,NDFRT,COSTAR,SNOMEDCT_US,DXP"
+}
+```
+
+Expanded detection will look for diseases, conditions, abnormalities, injuries, and procedures defined in the UMLS dictionary that ships with {{site.data.keyword.wh-acd_short}}.
```

Comment:
Looks like there are a number of places where this isn't handled or formatted correctly. {{site.data.keyword.wh-acd_short}} Maybe do a global search for the double curly bracket to find them all and fix.

Reply:
{{site.data.keyword.wh-acd_short}} should be replaced with IBM Clinical Data Annotator Service.